packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -160,10 +166,17 @@ function initDataviz(channel) {
     .d3Force('x', d3.forceX())
     .d3Force('y', d3.forceY())
     .d3Force('collide', d3.forceCollide(graph.nodeRelSize()))
-    .linkWidth(0.2)
+    .linkWidth(
+      () => parseFloat(model.style.lineWidth) || styleFallback.lineWidth
+    )
     .linkDirectionalParticles(1)
     .linkDirectionalParticleWidth(link =>
-      getLinkState(link, model) === 'highlighted' ? 1 : 0
+      getLinkState(link, model) === 'highlighted'
+        ? Math.max(
```

Comment:
what are you trying to convey here?
not sure this would achieve the desired particle effect, wouldn't it create particles that are as big as the line (so, invisible from a user's POV)?
Maybe we can make this an explicit setting, or computing it from `lineWidth`?

Reply:
I didn't dig deep enough into how force-graph is implemented but it seems that when the line width is < 1, then particle width needs to be quite large relatively to be visible - but once line width is > 1 then setting particleWidth = lineWidth produced particles slightly wider than the line. But I like your idea of having it explicit - leaving people to play with the setting to achieve their preferred look. I'll udate this too!